tests/test_live_validation.c

```diff
@@ -57,7 +59,7 @@ int main(void)
 {
 	/* create a TCP transport socket */
 	struct tr_socket tr_tcp;
-	struct tr_tcp_config tcp_config = {RPKI_CACHE_HOST, RPKI_CACHE_POST, NULL};
+	struct tr_tcp_config tcp_config = { (char *)RPKI_CACHE_HOST, (char *)RPKI_CACHE_POST, NULL};
```

Comment:
I'm not sure casting away const is the best fix here.
Copying the string would work.
@smlng your opinion?

Reply:
After RL discussion with @smlng  we think it would be best to declare `RPKI_CACHE_HOST` and `RPKI_CACHE_POST` as static variables (and maybe fix the typo while touching them …)